frontend/packages/dev-console/integration-tests/features/topology/horizonta_pod_autoscaling/HPA_Topology_&_Sidebar.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Add HPA action and topology sidebar modifications
+	As a user, I want to add a HPA to a workload
+ 
+Background:
+    Given user is in developer perspective
+    And user has created a deployment/deployment-config 
+    And user has assigned values to cpu and memory value in resource limit section of advanced option
+    And user has a workload with HPA assigned to it
+
+@regression @manual
+Scenario: Changes due to HPA in Workload Sidebar
```

Comment:
We should also check that there is also no `set pod count` in the actions.

Reply:
Yeah, I'm checking for `Edit Pod Count` option in the action menu

Reply:
Must have missed it.